operator-common/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlServerParameters.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource.cruisecontrol;
+
+import java.util.concurrent.TimeUnit;
+
+public enum CruiseControlServerParameters {
```

Comment:
can we change in `CruiseControlConfigurationParameters`? wdyt @tomncooper ?

Reply:
Agreed. That is a better name.

Reply:
Resolved